src/calculateRentalCost.js

```diff
@@ -29,6 +29,23 @@
 
 function calculateRentalCost(days) {
   // write code here
+  let discount = 0;
+  const discountLevel1 = 20;
+  const discountLevel2 = 50;
+  const daysAmount = days;
+
+  switch (true) {
```

Comment:
`switch` is usually used when there is a limited set of specific values that we want to compare to. Here, we have range of days, it's better to use `if`.